app/models/article.rb

```diff
@@ -1,3 +1,6 @@
 class Article < ApplicationRecord
   validates_presence_of :title, :content
+  validates :title, length: { minimum: 3, maximum: 50 }
```

Comment:
Code repetition, either make it oneliner or remove the validates_presence_of and add presence validation to the separate attributes.

Reply:
Of course!